ocs_samples/library_samples/Python3/setup.py

```diff
@@ -15,8 +15,8 @@
     url="https://github.com/osisoft/OSI-Samples",
     packages=setuptools.find_packages(),    
     install_requires=[
-        'requests==2.21.0',
-        'python-dateutil==2.8.0'
+        'requests>=2.21.0',
```

Comment:
Our policy with this is that we want to use the latest, but don't want to open it to unknown releases, which can include unwanted downstream dependencies or potential license changes

Reply:
Pinned to latest 2.22.0